### ISSUE TITLE
Validate repo cannot be checked out on Windows without errors

### ISSUE THREAD

**Issue by jpl-jengelke** [NASA-PDS-Incubator/validate#411]
## 🐛 Restricted file names in Validate Git repo prevent checkout on Windows
Test data contains files named "com4" at `./src/test/resources/github5/invalid/com4` and `./src/test/resources/github5/invalid/com4`. [These are prohibited by Windows](https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions). Also, see https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#win32-file-namespaces .

## 📜 To Reproduce
1. Open a terminal on a Windows workstation. 
2. Attempt a Git clone of `validate`, e.g. `git@github.com:NASA-PDS/validate.git`.
This yields: 
```
jengelke@<WS_ID> MINGW64 /c/sourcecode/source/pds
$ git clone git@github.com:NASA-PDS/validate.git
Cloning into 'validate'...
remote: Enumerating objects: 35826, done.
remote: Counting objects: 100% (7732/7732), done.
remote: Compressing objects: 100% (1655/1655), done.
remote: Total 35826 (delta 5853), reused 7389 (delta 5646), pack-reused 28094
Receiving objects: 100% (35826/35826), 61.13 MiB | 1.73 MiB/s, done.
Resolving deltas: 100% (27382/27382), done.
error: invalid path 'src/test/resources/github5/invalid/com4'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
jengelke@<WS_ID> MINGW64 /c/sourcecode/source/pds
$
```
3. `cd` to `validate` directory
4. `git status` shows numerous deleted files.
5. Executing recommended fixes yields: 
```
jengelke@<WS_ID> MINGW64 /c/sourcecode/source/pds/validate (main)
$ git restore --source=HEAD :/
error: invalid path 'src/test/resources/github5/invalid/com4'
error: invalid path 'src/test/resources/github5/invalid/lpt8'
error: invalid path 'src/test/resources/github5/invalid/lpt9.tab'
error: invalid path 'src/test/resources/github6/invalid/com4'
error: invalid path 'src/test/resources/github6/invalid/lpt8'
error: invalid path 'src/test/resources/github6/invalid/lpt9.tab'
jengelke@<WS_ID> MINGW64 /c/sourcecode/source/pds/validate (main)
$ 
```

## 🕵️ Expected behavior
The validate repo should clone seamlessly. Running `git status` in a newly checked out repo should not show numerous deleted files. 

## 📚 Version of Software Used
```
$ git --version
git version 2.33.0.windows.2
```
Validate repo: [latest](https://github.com/NASA-PDS/validate/commit/f384d07e70ee085b3a7972b861b8e8fd11687647)

## 🖥 System Info
- Edition	Windows 10 Enterprise
- Version	21H1
- Installed on	‎1/‎16/‎2021
- OS build	19043.1266
- Experience	Windows Feature Experience Pack 120.2212.3920.0

---

**Comment by jpl-jengelke** [comment]
This may be a good use case for making a separate repo for test data to decouple it. I could be wrong but the test data in resources doesn't seem used by unit tests but instead by Acceptance Testing.

**Comment by jordanpadams** [comment]
@jpl-jengelke 👍 totally agree. that is definitely our ideal future state :-)

**Comment by jpl-jengelke** [comment]
**Workaround:** 
- Install a `ZIP` file manager, like 7-zip
- Go to [releases](https://github.com/NASA-PDS/validate/releases)
- Click to expand `Assets`
  - Select `Source code (zip)` to download locally
- Open the downloaded repo in the `ZIP` file manager
- Delete the offending files
- Unpack (`unzip`) normally to the destination repository. 

**NOTE:** The product of the archive directory is named identically to the unpacked release binary directory, so if that was unpacked first then it will be overwritten or overloaded with files.

**Comment by jordanpadams** [comment]
resolved per 9e423288d60b1966d357e470e47420ccc98b11cd

@tdddblog when fixing the tests, you can refer to this ticket
